hoomd/md/force.py

```diff
@@ -172,6 +172,24 @@ def get_net_force(self,group):
 
         return (self.cpp_force.calcForceGroup(group.cpp_group).x, self.cpp_force.calcForceGroup(group.cpp_group).y, self.cpp_force.calcForceGroup(group.cpp_group).z)
 
+    def get_net_virial(self,group):
+        R""" Get the virial of a particle group.
+
+        Args:
+            group (:py:mod:`hoomd.group`): The particle group to query the virial for.
+
+        Returns:
+            The last computed virial for the members in the group.
+
+        Examples:
+
+            g = group.all()
+            virial = force.get_net_virial(g)
+        """
+        return self.cpp_force.calcVirialGroup(group.cpp_group)
```

Comment:
Convert return value a numpy array.

Generally user-facing API should not return C++ objects.

Reply:
I am betting here on the auto conversion of std::vector to numpy array.
Do you want an explicit conversion?